src/SSPL-1.0.xml

```diff
@@ -0,0 +1,643 @@
+          to the extent that it includes a convenient and prominently visible
+          feature that (1) displays an appropriate copyright notice, and (2)
+          tells the user that there is no warranty for the work (except to
+          the extent that warranties are provided), that licensees may convey
+          the work under this License, and how to view a copy of this License.
+          If the interface presents a list of user commands or options,
+          such as a menu, a prominent item in the list meets this criterion.</p>
+                </item>
+                <item>
+                    <bullet>1.</bullet>
+        Source Code.
+                    <br></br>
+        The "source code" for a work means the preferred
+        form of the work for making modifications to it.
+        "Object code" means any non-source form of a work.
+                    <p>
+          A "Standard Interface" means an interface that either is an official
+          standard defined by a recognized standards body, or, in the case
+          of interfaces specified for a particular programming language,
+          one that is widely used among developers working in that language.</p>
+                    <p>
+          The "System Libraries" of an executable work include anything, other
+          than the work as a whole, that (a) is included in the normal form
+          of packaging a Major Component, but which is not part of that Major
+          Component, and (b) serves only to enable use of the work with that
+          Major Component, or to implement a Standard Interface for which an
+          implementation is available to the public in source code form. A
+          "Major Component", in this context, means a major essential component
+          (kernel, window system, and so on) of the specific operating system
+          (if any) on which the executable work runs, or a compiler used to
+          produce the work, or an object code interpreter used to run it.</p>
+                    <p>
+          The "Corresponding Source" for a work in object code form means
+          all the source code needed to generate, install, and (for an
+          executable work) run the object code and to modify the work,
+          including scripts to control those activities. However, it does
+          not include the work's System Libraries, or general-purpose tools
+          or generally available free programs which are used unmodified in
+          performing those activities but which are not part of the work.
+          For example, Corresponding Source includes interface definition
+          files associated with source files for the work, and the source
+          code for shared libraries and dynamically linked subprograms
+          that the work is specifically designed to require, such as by
+          intimate data communication or control flow between those
+                        <br></br>
```

Comment:
This `<br></br>` looks like a stray line break mid-paragraph, can this come out?